smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/events/MessageMarshallableGenerator.kt

```diff
@@ -0,0 +1,204 @@
+package software.amazon.smithy.swift.codegen.events
```

Comment:
Other than this package statement, this file is unchanged from how it was in `aws-sdk-swift`.